.circleci/config.yml

```diff
@@ -13,28 +13,39 @@ commands:
             curl -sfSL --retry 5 --retry-delay 10 https://github.com/rust-lang-nursery/mdBook/releases/download/v0.4.2/mdbook-v0.4.2-x86_64-unknown-linux-gnu.tar.gz | tar xz
             echo 'export PATH="$HOME/.bin:$PATH"' >> $BASH_ENV
             popd
+  # Our policy for updating these rust versions is at https://github.com/mozilla/uniffi-rs/tree/main/docs/policies/rust-versions.md
+  # See also rust-toolchain.toml in the root of this repo, which is used to ensure the local version of rust matches.
+  prepare-rust-target-version:
+    steps:
+      # So long as this is executed after the checkout it will use the version specified in rust-toolchain.yaml
+      - run: rustup update
+  prepare-rust-min-version:
+    steps:
+      - run: rustup override set 1.47.0
+      - run: rustup update
+
 orbs:
   gh-pages: sugarshin/gh-pages@0.0.6
 jobs:
   Check Rust formatting:
     docker:
       # Our policy for updating this rust version is at https://github.com/mozilla/uniffi-rs/tree/main/docs/policies/rust-versions.md
       # See also rust-toolchain.toml in the root of this repo, which is used to ensure the local version of rust matches.
```

Comment:
This comment no longer refers to anything, as there is no "this rust version" in the lines that follow it.